src/patterns/components/offline-notification/example.hbs

```diff
@@ -0,0 +1,12 @@
+---
+name: Example
+sourceless: true
+hidden: true
+---
+
+<div class="Message Message--offline">
```

Comment:
Please create two separate patterns, one to show the _default_ look and feel and one for the offline appearance.